tests/ui/pages/catalog/catalog-page.ts

```diff
@@ -48,8 +51,24 @@ export default class CatalogPage extends BasePage {
     return new InputField(this.page, SELECTORS.searchBar);
   }
 
-  async searchByText(text: string) {
+  async fillSearchBar(text: string) {
     await this.searchBar.fill(text);
+  }
+
+  async confirmSearch() {
```

Comment:
What is the reason for dividing this single method:
```  
async searchByText(text: string) {
    await this.searchBar.fill(text);
    await this.page.locator(SELECTORS.searchBar).press('Enter');
}
```

Into 2 separate methods: `fillSearchBar` and `confirmSearch`
?

I see that you always use them together. So I don't see any reason for that.

Can we combine them together as we had earlier? I think it will make tests clearer and shorter

Reply:
Agree with @shilgam. For this reason, pls use the get `searchbarButton(){await new Button(this.page, SELECTORS.searchBar) } `to establish access to the button and then just add an action over it.

Reply:
I separated it because in the test "Should display full list when searching by empty string" I have to click in search button without any text in the search bar
Do you think that it is possible to do it in another way?

Reply:
The current implementation is much better:
```
  async fillSearchBar(text: string) {
    await this.searchBar.fill(text);
    await this.page.locator(SELECTORS.searchBar).press('Enter');
  }
```

Could u pls rename `fillSearchBar(text: string)` to `searchBy(text: string)` or similar?
Because the current name could be easily misunderstood in tests.